scripts/shuffle/vis.py

```diff
@@ -0,0 +1,116 @@
+# Copyright 2023 MosaicML Streaming authors
+# SPDX-License-Identifier: Apache-2.0
+
+"""Visualize different shuffling algorithms on fake shards."""
+
+import colorsys
+from argparse import ArgumentParser, Namespace
+
+import numpy as np
+
+from streaming.base.shuffle import algos, get_shuffle
+
+
+def parse_args() -> Namespace:
+    """Parse command-line arguments.
+
+    Returns:
+        Namespace: Command-line arguments.
+    """
+    args = ArgumentParser()
+    args.add_argument('--num_shards', type=int, default=100)
+    args.add_argument('--base_samples_per_shard', type=int, default=20)
+    args.add_argument('--num_canonical_nodes', type=int, default=16)
+    args.add_argument('--seed', type=int, default=0xD06E)
```

Comment:
seed default value !!! is it Hexa-demical value ?

Reply:
yep